spec/abstractops.html

```diff
@@ -114,19 +114,26 @@ <h1>Units</h1>
       Time is reckoned using multiple units.
       These units are listed in <emu-xref href="#table-temporal-units"></emu-xref>.
     </p>
+    <p>
+      A <dfn>Temporal unit</dfn> is a value listed in the "Value" column of <emu-xref href="#table-temporal-units"></emu-xref>.
+      A <dfn>calendar unit</dfn> is a Temporal unit for which IsCalendarUnit returns *true*, and a <dfn>non-calendar unit</dfn> is a Temporal unit for which IsCalendarUnit returns *false*.
+      A <dfn>date unit</dfn> is a Temporal unit for which the corresponding "Category" value in <emu-xref href="#table-temporal-units"></emu-xref> is ~date~, and a <dfn>time unit</dfn> is a Temporal unit for which the corresponding "Category" value is ~time~.
```

Comment:
_Temporal unit_, _calendar unit_, _date unit_, and _time unit_ all make sense to me, but I don't think I like _non-calendar unit_ because it also seems to cover e.g. kilograms. I think it would be better to replace the five uses of it with "a time unit or \~day~".